### PR TITLE
Activity 3 - Online Resume of Shaikram Abdulguro Jr - Advance

### PR DESCRIPTION
**Pull request by shaikram**
I used bootstrap template

### CODE REVIEW

File: shaikram_abdulguro/activity3/index.html [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+<html lang="en">
+    <head>
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+        <meta name="description" content="This is the summary of my skills"/>
+        <meta name="author" content="Aikee" />
+        <title>Resume | Shaikram Abdulguro Jr</title>
+        <link rel="icon" type="image/x-icon" href="assets/img/favicon.ico" />  
+        <!-- Google fonts-->
+        <link href="https://fonts.googleapis.com/css?family=Saira+Extra+Condensed:500,700" rel="stylesheet" type="text/css" />
```
Comment (mrmartincarlito) [inline shaikram_abdulguro/activity3/index.html:11]: If you can also download this asset

**Comment by mrmartincarlito** [comment]
May formatter kana ba na ginagamit ?

File: shaikram_abdulguro/activity3/index.html (hunk 1 above)
Comment (shaikram) [reply shaikram_abdulguro/activity3/index.html:11]: noted sir, i'll try

**Comment by shaikram** [comment]
> May formatter kana ba na ginagamit ?

wala pa sir, atom po kasi gamit ko dati nangangapa pako sa vscode

**Pull request merged**